<launch>

  <rosparam param="leftLinek1">[0, 0, 0, 0]</rosparam>
  <rosparam param="leftLinek2">[0, 0, 0, 0]</rosparam>
  <rosparam param="leftLinek3">[0, 0, 0, 0]</rosparam>

  <rosparam param="rightLinek1">[0, 0, 0, 0]</rosparam>
  <rosparam param="rightLinek2">[0, 0, 0, 0]</rosparam>
  <rosparam param="rightLinek3">[0, 0, 0, 0]</rosparam>

  <rosparam param="horizonLine">[0, 0, 0]</rosparam>


  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <remap from="usb_camera" to="camera" />
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="1280" />
    <param name="image_height" value="720" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
  </node>

  
  <node name="edges_Node" pkg="self_driving_car_thesis" type="edgesNode">
  </node>
  
  <node name="lines_Node" pkg="self_driving_car_thesis" type="linesNode">
  </node>
  
  
</launch>
